res/database/Default/user/kasi.xml

```diff
@@ -195,23 +222,28 @@
 			<title>
 				<de>Befreit den %Tier% %title%%Name%</de>
 				<en>Free the %Tier% %title%%Name%</en>
+				<pl>Uwolnić %Tier% %Name%</pl>
 			</title>
 			<description>
 				<de>Ein Kind verliebt sich beim Zoobesuch in den %Tier% %title%%Name%. Doch sieht dieser sehr traurig aus. So tut das Kind alles in seiner Macht stehende, um ihn zu befreien.</de>
 				<en>A child falls in love with the %Tier% %title%%Name% while visiting the zoo. But it looks very sad. So the child does everything in its power to free the animal.</en>
+				<pl>Dziecko zakochuje się w zwierzątku podczas wizyty w zoo. Wygląda ono jednak na bardzo smutne. Dziecko robi więc wszystko, co w jego mocy, by uwolnić %Name%.</pl>
 			</description>
 			<variables>
-				<tier>
+				<Tier><!--tier changed to Tier.  -->
 					<de>Affen|Nacktmull|Goldfisch|Löwen|Tiger|Hasen|Panda</de>
 					<en>monkey|naked mole rat|goldfish|lion|tiger|rabbit|panda</en>
-				</tier>
-				<name>
+					<pl>małpę|surykatkę|delfina|lwa|tygrysa|zająca|pandę</pl>
+				</Tier>
+				<Name><!--name changed to Name.  -->
 					<de>Jingels|Wollo|Hubert|Billy|Fluffy|Bobo|Flocke</de>
 					<en>Jingels|Wollo|Hubert|Billy|Fluffy|Bobo|Flake</en>
-				</name>
+					<pl>Jingela|Wollo|Huberta|Billiego|Fluffiego|Bobo|Neo</pl>
+				</Name>
 				<title>
 					<de>Mister |Little |Big |Mac</de>
 					<en>Mister |Little |Big |Mac</en>
+					<!--In Polish, not required. I had to rewrite it to make it grammatically ok.  -->
```

Comment:
May be OK, but the "title" actually modifies the name. MacFluffy vs. Mister Fluffy.

Reply:
I'm not using title.